src/cbmc/cbmc_parse_options.cpp

```diff
@@ -13,6 +13,7 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <fstream>
 #include <cstdlib> // exit()
+#include <iomanip>
 #include <iostream>
```

Comment:
Is the extra header needed?

Reply:
My apologies, that was a left-over that I had forgotten to clean up. It's gone now.